buffer_period.go

```diff
@@ -135,16 +134,14 @@ func (t *timer) stop() {
 
 // tick updates the minimum buffer timer and returns whether the timer
 // was active.
```

Comment:
minor nit: looks like you can remove the content around what is returned

Reply:
thanks!